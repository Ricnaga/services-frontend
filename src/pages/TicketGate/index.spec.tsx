import { render } from '@testing-library/react';
import { TicketGate } from '.';

describe('TICKET GATE SCREEN', () => {
  it('should be able to render screen', () => {
    const { container } = render(<TicketGate />);

    expect(container).toMatchSnapshot();
  });
});
