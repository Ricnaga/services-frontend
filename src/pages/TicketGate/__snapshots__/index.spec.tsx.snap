// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TICKET GATE SCREEN should be able to render screen 1`] = `
<div>
  <div
    class="mt-4 w-50 container-fluid"
  >
    <div
      class="card"
    >
      <h3
        class="card-header"
      >
        Insira suas credenciais
      </h3>
      <div
        class="px-4 card-body"
      >
        <form
          class=""
        >
          <div
            class="mb-3 input-group"
          >
            <input
              class="form-control"
              name="id"
              placeholder="Inserir número do crachá"
              value=""
            />
            <button
              class="btn btn-outline-warning btn-lg"
              type="submit"
            >
              Entrar
            </button>
          </div>
        </form>
      </div>
    </div>
  </div>
</div>
`;
